biz.aQute.bndlib/src/aQute/bnd/differ/DiffPluginImpl.java

```diff
@@ -100,9 +100,8 @@ public Tree tree(File newer) throws Exception {
 	 * @see aQute.bnd.service.diff.Differ#tree(aQute.bnd.osgi.Jar)
 	 */
 	public Tree tree(Jar newer) throws Exception {
-		Analyzer anewer = new Analyzer();
+		Analyzer anewer = new Analyzer(newer);
 		try {
-			anewer.setJar(newer);
 			return tree(anewer);
 		} finally {
 			anewer.setJar((Jar) null);
```

Comment:
But I suppose the set to null is to avoid closing the Jar when closing the analyzer. You are right. This method should not close the jar.